Code.js

```diff
@@ -0,0 +1,65 @@
+function onInstall() {
+  onOpen();
+}
+
+function onOpen() {
+  var ui = SpreadsheetApp.getUi() // Or DocumentApp or SlidesApp or FormApp.
+      ui.createMenu('Timesheet')
+      .addItem('Update Project Time', 'showProject')
+      .addToUi();
+}
+
+function doPost(e) {
+  Logger.log(e); 
+}
+
+function showProject() {
+  var html = HtmlService.createHtmlOutputFromFile('updateProject')
+      .setTitle('Update Timesheet')
+      .setWidth(300);
+  SpreadsheetApp.getUi() // Or DocumentApp or SlidesApp or FormApp
+      .showSidebar(html);
+}
+
+// send user input to Google Spreadsheet
+function sendAll(datas) {
+  var sheet = SpreadsheetApp.openById('1dRAXZRHMgfR_oX1ZF1RFeonUi7ZPk_wbRf7Jx0UvCjE').getActiveSheet(); // ID for Timesheet
```

Comment:
needs to be configurable.

use PropertiesService to store as a named property in the document.

document the property configuration in README.md